### DISCUSSION TITLE
Dhizuku API

### DISCUSSION THREAD

**Discussion by anransuibuxing** [BinTianqi/OwnDroid#23] [Q&A] [answered]
希望加入对 Dhizuku 权限的支持(🙏ˊᗜˋ*)

**Comment by BinTianqi** [comment]
看 Issue [#9](https://github.com/BinTianqi/OwnDroid/issues/9#issuecomment-2080340135)

如果真的有需要用Dhizuku那就用[DAX](https://github.com/BinTianqi/OwnDroid/issues/9#issuecomment-2066138671)模块

**Reply by anransuibuxing** [reply]
大佬有没有兴趣加入鄙人的设备管理员讨论群，为群友们指点迷津呢？
QQ：914579017

**Comment by BinTianqi** [comment, marked as answer]
#61
